convert_case.py

```diff
@@ -1,13 +1,14 @@
-message = input("Please write the string you want to case-convert: ")
-option = input("Please choose from the following options: \n \
+message = str(input("Please write the string you want to case-convert: "))
+option = str(input("Please choose from the following options: \n \
   L for lower case \n \
   U for upper case \n \
   T for title case \n \
   C for capitalized case \n \
   S for swap case \n \
-  ______________________________ \n \
+  OU for upper case by even ordering \n \
```

Comment:
Nice one! 👍 
```suggestion
  A for alternate case \n \
```

I suggest renaming the option from `OU` to just `A` that way referring to alternate caps. Please see the following links:

[https://en.wikipedia.org/wiki/Alternating_caps](https://en.wikipedia.org/wiki/Alternating_caps)
[https://convertcase.net/](https://convertcase.net/)